frontend/src/App/index.js

```diff
@@ -34,18 +39,21 @@ class App extends React.Component {
           <Router history={history}>
             <div>
               <Header />
-              <Switch>
-                <Route exact path="/" component={PaginaPrincipal} />
-                <Route path="/login" component={LoginPage} />
-                <Route path="/register" component={RegisterPage} />
-                <PrivateRoute exact path="/home_user" component={HomePage} />
-                <Redirect from="*" to="/" />
-              </Switch>
+              <div class="main">
+                <Switch>
+                  <Route exact path="/" component={PaginaPrincipal} />
+                  <Route path="/login" component={LoginPage} />
+                  <Route path="/register" component={RegisterPage} />
+                  <Route path="/detail" component={Detail} />
```

Comment:
¿Esta ruta no debería ser privada?

Reply:
Si se pone como ruta privada, no carga cuando intentas acceder a la dirección haciendo "click" en "Sign In", por ejemplo.